docs/API.md

```diff
@@ -170,7 +170,7 @@ Basic data type: object
 Properties:
 
 * `"ns"`: A *domain name*, required.
-* `"ip"`: An *IP address*, IPv4 or IPv6 , optional.
+* `"ip"`: An *IP address* (IPv4 or IPv6), optional. (no default value)
```

Comment:
"no default value" is synonymous to "default: unset", right? Could you make the update to consistently use either one?

Reply:
done